scripts/sct_detect_pmj.py

```diff
@@ -58,6 +58,10 @@ def get_parser():
                         mandatory=False,
                         example=["0", "1"],
                         default_value="0")
+    parser.add_option(name="-igt",
+                      type_value="image_nifti",
+                      description="File name of ground-truth PMJ (single voxel).",
+                      mandatory=False)
```

Comment:
Not used?

Reply:
nope!

context: in the past we evaluated the quality of segmentation by inputing a ground truth segmentation, and it would output the Dice-- that was bad practice (it should have been externalized instead of cluttering this CLI, and also it was only dedicated to the devs)